### DISCUSSION TITLE
coder workspace - Workspace is unhealthy

### DISCUSSION THREAD

**Discussion by PecTheFabricator** [coder/coder#15498] [General]
Hi team,

I'm trying to run a docker container/Kubernetes pods in my workspace, however, I'm facing the issue **Workspace is unhealthy**. 
To make sure the container/pod is running, I have attached the container/pod shell and I was able to join in. One thing I have noticed is that the code-server bin is not being downloaded to the temp folder. So if I run code-server inside the container, it doesn't work.

Note: If I use the default image in the workspace (the buildenv one) it works as expected.

**Container Image**: quay.io/jupyter/base-notebook:latest
**Template**: https://github.com/coder/coder/tree/main/examples/templates/docker
![image](https://github.com/user-attachments/assets/4098deec-a59f-47be-a39f-c962266bd423)

Thanks

**Comment by matifali** [comment]
Can you exec into the container and make sure it has `curl` installed? Coder needs `curl` to download the coder agent binary in the workspace.

**Reply by PecTheFabricator** [reply]
Sorry for later reply. I will check that and go back

**Reply by PecTheFabricator** [reply]
As asked, I have added the curl package and I have also added the user coder, however, the issue persist. I can execute the container, but still not coder server inside the container

**Reply by matifali** [reply]
Could you share the logs from `/tmp/coder-agent.logs` after execing into the container?

**Reply by PecTheFabricator** [reply]
> Could you share the logs from `/tmp/coder-agent.logs` after execing into the container?

Its is very stranger, there is nothing related to the coder inside the tmp
